### ISSUE TITLE
linktype="examples"のリソースの扱いについて

### ISSUE THREAD

**Issue by momdo** [waic/wcag21#267]
https://github.com/waic/wcag20/issues/175#issuecomment-442746504

> `examples`に関しては、WCAG 2.1でも同じものがあるようです。リンクテキストを和訳して、（将来の話ですが）リソースもW3Cではなく自前で用意するとなると、リンク先も手を加える必要があるのかもしれません。
>
> - https://www.w3.org/WAI/WCAG21/Techniques/failures/F91#examples
> - https://www.w3.org/WAI/WCAG21/working-examples/failure-td-not-th/

**Comment by bakera** [comment]
コード例は翻訳しない方針なので、日本語にして持ってくる必要はない
W3Cのものに直接リンクで良い

リンクテキストは日本語訳して「英語」と書いても良いけれども、2.0のほうではリンクテキストも英語のままだった。
こちらもリンクテキストを英語のままにする方向で良いのではないか。

**Comment by momdo** [comment]
> コード例は翻訳しない方針なので、日本語にして持ってくる必要はない
W3Cのものに直接リンクで良い
>
> リンクテキストは日本語訳して「英語」と書いても良いけれども、2.0のほうではリンクテキストも英語のままだった。
こちらもリンクテキストを英語のままにする方向で良いのではないか。

という方針に決まっていた（というよりも、このissueを閉じ忘れていた）ので、Closeとしますね。
